docs/features/multiple-networks/multi-homing.md

```diff
@@ -201,9 +276,10 @@ spec:
 ```
 
 ### Setting static IP addresses on a pod
+
```

Comment:
... why ?

Reply:
Changed as part of [comment](https://github.com/ovn-org/ovn-kubernetes/pull/4537/files#r1745234474)